### PR TITLE
Update for stateless (part 2)

### PR DESCRIPTION
**Pull request by thewilkybarkid**
Matches https://github.com/elifesciences/recommendations/pull/83.

### CODE REVIEW

File: salt/recommendations/init.sls [hunk 1]
```diff
@@ -1,3 +1,26 @@
...
+              - /etc/init/goaws-init.conf
+              - /etc/init/recommendations-processes.conf
+              - /etc/init/recommendations-queue-watch.conf
+              - /etc/mysql/
+              - /etc/nginx/sites-enabled/api-dummy-recommendations.conf
+              - /home/{{ pillar.elife.deploy_user.username }}/.aws/
+              - /srv/api-dummy/
+              - /srv/recommendations/config/
+              - /srv/recommendations/var/cache/
+              - /usr/local/src/github.com/
```
Comment (thewilkybarkid) [inline salt/recommendations/init.sls]: Looks to only be Go stuff?

File: salt/recommendations/init.sls [hunk 2]
```diff
@@ -1,3 +1,26 @@
+recommendations-legacy-removal:
+    service.dead:
+        - names:
+              - goaws
```
Comment (thewilkybarkid) [inline salt/recommendations/init.sls]: Are there more processes on EC2 instances? I currently don't have access.

File: salt/example.top [hunk 3]
```diff
@@ -5,13 +5,5 @@ base:
         - elife.composer
         - elife.nginx
         - elife.nginx-php7
-        - elife.mysql-server
-        - elife.mysql-client
```
Comment (giorgiosironi) [inline salt/example.top]: probably api-dummy could stay, to quickly run tests?

File: salt/recommendations/init.sls (hunk 2 above)
Comment (giorgiosironi) [inline salt/recommendations/init.sls]: there is `recommendations-queue-watch`

File: salt/recommendations/init.sls (hunk 1 above)
Comment (giorgiosironi) [inline salt/recommendations/init.sls]: yes, it's the `GOPATH`/`GOROOT`

File: salt/recommendations/init.sls [hunk 4]
```diff
@@ -1,3 +1,26 @@
...
+              - golang-go
+              - mysql-client
+              - mysql-server
+              - python-mysqldb
+    file.absent:
+        - names:
+              - /etc/init/goaws-init.conf
+              - /etc/init/recommendations-processes.conf
+              - /etc/init/recommendations-queue-watch.conf
+              - /etc/mysql/
```
Comment (giorgiosironi) [inline salt/recommendations/init.sls]: by purging mysql-server `/etc/mysql/` could go away on its own

File: salt/example.top (hunk 3 above)
Comment (thewilkybarkid) [inline salt/example.top]: Could do, but means maintenance in keeping it up to date. Re-add if/when we need it?

Comment (giorgiosironi) [inline salt/example.top]: I thought it was needed, if there aren't tests leveraging it, then remove

**Pull request merged**